vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -178,5 +188,16 @@ def parse_base_item(item)
         :resource_version   => item.metadata.resourceVersion
       }
     end
+
+    HUMAN_SIZE_SUFFIXES = ["Ki", "Mi", "Gi", "Ti"]
```

Comment:
Suffixes should be `["KB", "MB", "GB", "TB"]`, right?  Or, are these the suffixes that are coming in from Kubernetes API?